parm/archive/master_enkf.yaml.j2

```diff
@@ -4,28 +4,6 @@
 {% set cycle_YMDH = current_cycle | to_YMDH %}
 {% set head = RUN + ".t" + cycle_HH + "z." %}
 
```

Comment:
```suggestion

{% if IAUFHRS is integer %}
    {% set IAUFHRS = [IAUFHRS] %}
{% endif %}

{% if IAUFHRS_ENKF is integer %}
    {% set IAUFHRS_ENKF = [IAUFHRS_ENKF] %}
{% endif %}

```

Reply:
I think this won't be necessary once I do the thing I intended to do, which is add a trailing comma to the single-digit IAUFHRS.